app/com/arpnetworking/metrics/portal/scheduling/JobExecutorActor.java

```diff
@@ -45,126 +45,137 @@
 public final class JobExecutorActor<T> extends AbstractActorWithTimers {
 
     private final Injector _injector;
+    private final Organization _organization;
     private final JobRef<T> _jobRef;
     private final Clock _clock;
+    private PeriodicMetrics _periodicMetrics;
 
     /**
      * Props factory.
      *
      * @param <T> The type of result produced by the {@link JobRef}'s job.
      * @param injector The Guice injector to use to load the {@link JobRepository} referenced by the {@link JobRef}.
+     * @param organization The {@link Organization} that the job-to-be-run belongs to.
      * @param jobRef The job to intermittently execute.
      * @return A new props to create this actor.
      */
-    public static <T> Props props(final Injector injector, final JobRef<T> jobRef) {
-        return props(injector, jobRef, Clock.systemUTC());
+    public static <T> Props props(final Injector injector, final Organization organization, final JobRef<T> jobRef) {
+        return props(injector, organization, jobRef, Clock.systemUTC());
     }
 
     /**
      * Props factory.
      *
      * @param <T> The type of result produced by the {@link JobRef}'s job.
      * @param injector The Guice injector to use to load the {@link JobRepository} referenced by the {@link JobRef}.
+     * @param organization The {@link Organization} that the job-to-be-run belongs to.
      * @param jobRef The job to intermittently execute.
      * @param clock The clock the scheduler will use, when it ticks, to determine whether it's time to run the next job(s) yet.
      * @return A new props to create this actor.
      */
-    protected static <T> Props props(final Injector injector, final JobRef<T> jobRef, final Clock clock) {
-        return Props.create(JobExecutorActor.class, () -> new JobExecutorActor<>(injector, jobRef, clock));
+    protected static <T> Props props(final Injector injector, final Organization organization, final JobRef<T> jobRef, final Clock clock) {
+        return Props.create(JobExecutorActor.class, () -> new JobExecutorActor<>(injector, organization, jobRef, clock));
     }
 
-    private JobExecutorActor(final Injector injector, final JobRef<T> jobRef, final Clock clock) {
+    private JobExecutorActor(final Injector injector, final Organization organization, final JobRef<T> jobRef, final Clock clock) {
```

Comment:
Not understanding why the organization is pulled out of the JobRef; it doesn't seem to matter. However, a JobRef is incomplete without an organization as far as I understand the concept. We should sync with @BrandonArp .